app/views/hyrax/base/_show_actions.html.erb

```diff
@@ -1,7 +1,7 @@
-<%# OVERRIDE Hyrax v5.0.0rc2 Adjust permission checks and add analytics button %>
+<%# OVERRIDE Hyrax v5.0.0rc2 Adjust permission checks and add analytics button and add ENV to control turbolinks %>
 <div class="show-actions">
   <% if presenter.editor? %>
-      <%= link_to t('.edit'), edit_polymorphic_path([main_app, presenter]), class: 'btn btn-secondary' %>
+      <%= link_to t('.edit'), edit_polymorphic_path([main_app, presenter]), class: 'btn btn-secondary', data: { turbolinks: ENV.fetch('TURBOLINKS_FOR_VALKYRIE', 'true') } %>
```

Comment:
Question: does `ENV.fetch('TURBOLINKS_FOR_VALKYRIE')` return `false`?  Or `"false"`? (string or boolean)

Reply:
it does return a string